frontend/src/pages/add-blog.tsx

```diff
@@ -89,95 +89,105 @@ function AddBlog() {
   };
   const navigate = useNavigate();
   return (
-    <div className="font-[Poppins] p-4 px-16 bg-white ">
-      <div className="flex justify-start items-center mb-4">
-        <div className=" text-black cursor-pointer w-fit  text-base md:text-lg lg:text-2xl">
-        <img src={blackleftarrow}  style={{height:20, width:40}}onClick={() => navigate(-1)} />
-        </div>
-        <h2 className="text-sm md:text-lg  lg:text-2xl font-bold ml-4">Create Post</h2>
-      </div>
+    <div className=" w-full min-h-screen bg-dark-bg">
+      <div className="w-full min-h-screen ">
+        <div className="font-[Poppins] p-4 px-16 bg-white dark:bg-dark-bg ">
```

Comment:
Do we actually need this extra divs ? can't we simplify this

Reply:
I will recheck and I will try to reduse div

Reply:
@krishnaacharyaa  variable name changed, add blog and detail page completed . only home page remaning will complete.

Reply:
Sure @SankalpHaritash21 Thank you for the update